tls/s2n_connection.h

```diff
@@ -366,6 +366,7 @@ struct s2n_connection {
     struct s2n_blob client_ticket;
     uint32_t ticket_lifetime_hint;
     struct s2n_ticket_fields tls13_ticket_fields;
+    uint8_t resume_protocol_version;
```

Comment:
Can you add a comment for this field or maybe preface your explanation of why `s2n_resume_protocol_version` exists with just a brief comment on what it is?